docs/Training-Behavioral-Cloning.md

```diff
@@ -1,30 +0,0 @@
-# Training with Behavioral Cloning
-
-There are a variety of possible imitation learning algorithms which can
-be used, the simplest one of them is Behavioral Cloning. It works by collecting
-demonstrations from a teacher, and then simply uses them to directly learn a
-policy, in the same way the supervised learning for image classification
-or other traditional Machine Learning tasks work.
-
-## Offline Training
```

Comment:
This section seems like it should be moved somewhere else (maybe to its own file?).

Reply:
Nm, guess it's already in Training-Imitation-Learning.md